### PR TITLE
Updated title

### PR DESCRIPTION
**Pull request by Pratichhya**
@megha-devaraju @Kampen1988 updated title based on Andras suggestions. Please let us know if addition changes in text are needed

### CODE REVIEW

File: openeo/UDF.ipynb [hunk 1]
```diff
@@ -230,9 +230,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "OpenEO",
+   "display_name": "openeo_py",
    "language": "python",
-   "name": "openeo"
```
Comment (soxofaan) [inline openeo/UDF.ipynb:235]: Note that changing the kernel name might trigger a "Select kernel" popup to users that open this notebook on CDSE jupyterlab when the `name` does not exist there.

In this case kernel `python3`  does exist on CDSE jupyterlab, but it's the base python and  doesn't have any of the importabt packages, like  `openeo` installed I think

Comment (Pratichhya) [reply openeo/UDF.ipynb:235]: Ahh sorry for confusion @soxofaan. Changes were made locally and didn't realise that change in markdown could also trigger it

Comment (soxofaan) [reply openeo/UDF.ipynb:235]: apparently this is documented here: https://github.com/eu-cdse/notebook-samples#kernel-spec-metadata

Comment (Pratichhya) [reply openeo/UDF.ipynb:235]: yes updated

Comment (soxofaan) [reply openeo/UDF.ipynb:235]: FYI: under #22 I added a kernel check script that now runs as pre-commit and github actions workflow

**Comment by Pratichhya** [comment]
@megha-devaraju @Kampen1988 a simple reminder :)

**Pull request merged**